### PR TITLE
added contributors section on team page

### LINKED ISSUE #3163: Missing contributors  section in team section
**Actual Behaviour**
There is no contributors section

**Would you like to work on the issue?**
YES , if the contributors section is required

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3163

Changes:

added contributors section on team page

Demo Link: https://pr-3223-fossasia-susi-web-chat.surge.sh/team

Screenshots of the change: 

Hnet-image (3)

### CODE REVIEW

**Comment by anishagg17** [comment]
@akshatnitd  please review this pr it has nothing related to server and hence you can clearly view the changes

**Comment by anishagg17** [comment]
> Pls update your branch.

done bro

**Comment by mariobehling** [comment]
This is a very nice idea, but it only shows the contributors of the susi.ai repo itself. Is it actually possible to use the Github API across all SUSI.AI repos? Otherwise it does not seem to be suitable due to the current limitation.

**Comment by anishagg17** [comment]
@mariobehling   It can also be  implement by using multiple apis (all susi.ai apis) . But I think that it should only contain contributors to web_client

**Comment by anishagg17** [comment]
>Is it actually possible to use the Github API across all SUSI.AI repos?
 Yes this api  can be used for all

File: src/components/About/Team/index.js [hunk 1]
```diff
@@ -133,18 +134,61 @@ const Description = styled.div`
   padding: 1rem;
 `;
 
+const contributorsLink =
+  'https://api.github.com/repos/fossasia/susi.ai/contributors';
+
```
Comment (snitin315) [inline src/components/About/Team/index.js]: use api from all susi.ai repositories and add those contributors as well.

Comment (anishagg17) [reply src/components/About/Team/index.js]: Done bro

**Comment by anishagg17** [comment]
@snitin315 all changes have been made .

File: src/components/About/Team/index.js [hunk 2]
```diff
@@ -133,18 +131,81 @@ const Description = styled.div`
...
+    contributors: [],
+  };
+
+  async componentDidMount() {
     // Adding title tag to page
     document.title =
       'Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
+
+    let contributors = [];
+    let response = await axios.get(contributorsWebLink);
```
Comment (akshatnitd) [inline src/components/About/Team/index.js]: Please move all API calls to apis file. It internally uses `axios` to make API calls.

File: src/components/About/Team/index.js [hunk 3]
```diff
@@ -133,18 +131,81 @@ const Description = styled.div`
...
+  'https://api.github.com/repos/fossasia/susi_server/contributors';
 export default class Team extends Component {
   static propTypes = {
     history: PropTypes.object,
     location: PropTypes.object,
   };
 
-  componentDidMount() {
+  state = {
+    contributors: [],
```
Comment (akshatnitd) [inline src/components/About/Team/index.js]: Also, can you use hooks here and convert the component to a functional one?

Comment (anishagg17) [reply src/components/About/Team/index.js]: Sir , I have also opened an issue #3431  to convert the class based components to  functional  components

**Comment by anishagg17** [comment]
@akshatnitd all changes have been done

**Comment by Dilshaad21** [comment]
@anishagg17 Squash your commits

**Comment by anishagg17** [comment]
@Dilshaad21 all changes have been made

**Comment by anishagg17** [comment]
@akshatnitd please review this pr

File: src/apis/index.js [hunk 4]
```diff
@@ -100,6 +100,37 @@ export function fetchDevices(payload) {
   return ajax.get(url, payload, { shouldCamelizeKeys: false });
 }
 
+export async function getContributors() {
+  const contributorsWebLink =
```
Comment (akshatnitd) [inline src/apis/index.js]: There are otherr SUSI.AI repos as well like `susi_linux`, etc. Please make a list of all active SUSI.AI repos and then show contributors in all of them.

Also, here code is being repetative. Avoid that. Also when you need to make multiple calls, use `Promise.all`

**Pull request merged**